huxley/www/js/stores/DelegateStore.js

```diff
@@ -78,6 +78,7 @@ class DelegateStore extends Store {
         break;
       case ActionConstants.UPDATE_DELEGATES:
         this.updateDelegates(action.schoolID, action.delegates);
+        break;
```

Comment:
I think the test tests whether the store was updated, not whether the store was updated and a change emitted